### PR TITLE
[FV0] Trigger in digits

### PR DESCRIPTION
**Pull request by arvindkhuntia**
This PR include 
(1) Implementaion of QED events for digitisation [first version]
(2) Trigger for FV0 detector

### CODE REVIEW

File: Detectors/FIT/FV0/simulation/include/FV0Simulation/Digitizer.h [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include <CommonDataFormat/InteractionRecord.h>
 #include <array>
 #include <vector>
+#include <TH1D.h>
```
Comment (mslupeck) [inline Detectors/FIT/FV0/simulation/include/FV0Simulation/Digitizer.h]: Is TH1D needed, or has it remained after testing? If possible remove please.

File: Detectors/FIT/FV0/simulation/src/Digitizer.cxx [hunk 2]
```diff
@@ -198,27 +195,49 @@ void Digitizer::storeBC(const BCCache& bc, std::vector<o2::fv0::BCData>& digitsB
...
     ++nStored;
+    //---trigger---
+    if (iPmt < 25)
+      nSignalInner++;
+    else
+      nSignalOuter++;
+    //LOG(INFO) <<" cfd "<< cfdZero;
   }
+  if (nStored < 1)
+    return;
```
Comment (mslupeck) [inline Detectors/FIT/FV0/simulation/src/Digitizer.cxx]: Add braces around if/else (3 missing). It improves readability and makes the fullCI checker happy

File: Steer/DigitizerWorkflow/src/FV0DigitizerSpec.cxx [hunk 3]
```diff
@@ -29,6 +29,7 @@
 #include "SimulationDataFormat/MCCompLabel.h"
 #include "DetectorsBase/BaseDPLDigitizer.h"
 #include <TFile.h>
+#include <TH1D.h>
```
Comment (mslupeck) [inline Steer/DigitizerWorkflow/src/FV0DigitizerSpec.cxx]: Remove if possible

File: Detectors/CTF/test/test_ctf_io_ft0.cxx [hunk 4]
```diff
@@ -39,7 +39,7 @@ BOOST_AUTO_TEST_CASE(CTFTest)
     auto start = channels.size();
     int32_t tMeanA = 0, tMeanC = 0;
     int32_t ampTotA = 0, ampTotC = 0;
-    Triggers trig;
+    Triggers trig; // TODO: Actual values are not set
```
Comment (mslupeck) [inline Detectors/CTF/test/test_ctf_io_ft0.cxx]: Why did you modify FT0 code? Was it intentional?

**Review by mslupeck — CHANGES_REQUESTED** [review]
Thanks @arvindkhuntia 
Sorry, it took a while to find time to go through your code. Could you consider making the proposed cosmetic changes?

File: Detectors/FIT/FV0/simulation/include/FV0Simulation/Digitizer.h (hunk 1 above)
Comment (arvindkhuntia) [reply Detectors/FIT/FV0/simulation/include/FV0Simulation/Digitizer.h]: Yes Maciej, it remained after testing. Now removed it. Thank you.

File: Detectors/FIT/FV0/simulation/src/Digitizer.cxx (hunk 2 above)
Comment (arvindkhuntia) [reply Detectors/FIT/FV0/simulation/src/Digitizer.cxx]: Done

File: Steer/DigitizerWorkflow/src/FV0DigitizerSpec.cxx (hunk 3 above)
Comment (arvindkhuntia) [reply Steer/DigitizerWorkflow/src/FV0DigitizerSpec.cxx]: Removed

File: Detectors/CTF/test/test_ctf_io_ft0.cxx (hunk 4 above)
Comment (arvindkhuntia) [reply Detectors/CTF/test/test_ctf_io_ft0.cxx]: This line is removed now. It was a mistake.
Thanks for this.

**Review by mslupeck — APPROVED** [review]
For me it's OK to be merged

**Review by AllaMaevskaya — APPROVED** [review]
For me OK

**Pull request merged**